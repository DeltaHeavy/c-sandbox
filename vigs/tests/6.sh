#!/bin/bash

echo -n "Test 6: ftable skip...."
./ftable -s 2 tests/6.in tests/6.testout
diff tests/6.testout tests/6.out >/dev/null 2>&1
ret=$?
rm tests/6.testout
if [ $ret -ne 0 ]
then
    echo -e "\033[31mFAIL\033[0m"
    false
else 
    echo -e "\033[32mPASS\033[0m"
    true
fi
