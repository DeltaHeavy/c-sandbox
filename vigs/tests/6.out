Total chars: 11
A:         0 (  0.00%) 
B:         2 ( 18.18%) *******************
C:         0 (  0.00%) 
D:         0 (  0.00%) 
E:         2 ( 18.18%) *******************
F:         0 (  0.00%) 
G:         0 (  0.00%) 
H:         0 (  0.00%) 
I:         0 (  0.00%) 
J:         0 (  0.00%) 
K:         0 (  0.00%) 
L:         0 (  0.00%) 
M:         0 (  0.00%) 
N:         1 (  9.09%) **********
O:         3 ( 27.27%) ****************************
P:         0 (  0.00%) 
Q:         0 (  0.00%) 
R:         1 (  9.09%) **********
S:         0 (  0.00%) 
T:         2 ( 18.18%) *******************
U:         0 (  0.00%) 
V:         0 (  0.00%) 
W:         0 (  0.00%) 
X:         0 (  0.00%) 
Y:         0 (  0.00%) 
Z:         0 (  0.00%) 
